package.json

```diff
@@ -30,11 +31,11 @@
     "@types/node": "17.0.43",
     "@types/react": "18.2.0",
     "@types/react-dom": "18.2.0",
-    "eslint": "8.17.0",
+    "@types/papaparse": "5.3.14",
+    "@typescript-eslint/eslint-plugin": "7.7.1",
```

Comment:
@SarahDonvito this upgrade sets node version to a version higher than 18.11.
Change readme to inform other dev to install one of the two LTS node versions available